src/app/cities/page.tsx

```diff
@@ -16,26 +16,55 @@ function getCitySize(population: number): '🏡Small' | '🏙️Middle-Size' | '
   return '🌆Big'
 }
 
+function getWinterSeverity(winter: string) {
```

Comment:
Os parâmetros para considerar um winter mild, cold ... ja estão definidos? Se sim, quais são? 
Dias do ano com temperatura negativa? Chega abaixo de -20C?

Reply:
Seria bom a gente começa a documentar os critérios... Como: o que eh uma big city, costOfLiving high....

Reply:
o criterio foi dado pelo chatGPT hahah

Reply:
mas sim, seria uma boa documentar o cost of living